libs/application/templates/secondary-school/src/fields/overview/CustodianOverview.tsx

```
@@ -110,10 +124,20 @@ export const CustodianOverview: FC<FieldBaseProps> = ({
             {showOtherContacts &&
               otherContacts.map((otherContact, index) => (
                 <GridColumn span="1/2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)